tests/core/test_core.py

```diff
@@ -125,12 +127,15 @@ def test_core_tiny_database(tmp_path):
     """Verify that we can generate core tables with some minimal data filled in"""
     testbed = testbed_utils.LocalTestbed(tmp_path)
     # Just add bare resources, with minimal data
+    testbed.add_allergy_intolerance("AllA")
```

Comment:
I can see myself getting confused by this abbreviation at some point (I already have a hard enough time holding the abstractions against the abbreviations here as is, but this one parses as "All A" too easily). Can it be more verbose?

Reply:
Updated :+1: